src/sailjail.c

```diff
@@ -384,36 +370,22 @@ int main(int argc, char* argv[])
      * options to overwrite those specified in the config file.
      */
     if (ok) {
-        if (args.conf_file) {
-            /* Config file was specified on the command line */
-            ok = jail_conf_load(conf, args.conf_file, &error);
-        } else {
-            /* The default config file may be (and usually is) missing */
-            if (g_file_test(DEFAULT_CONF_FILE, G_FILE_TEST_EXISTS)) {
-                jail_conf_load(conf, DEFAULT_CONF_FILE, NULL);
-            }
+        /* The config file may be (and usually is) missing */
+        if (g_file_test(DEFAULT_CONF_FILE, G_FILE_TEST_EXISTS)) {
+            jail_conf_load(conf, DEFAULT_CONF_FILE, NULL);
         }
 
-        if (ok) {
-            /* Allocate new parser */
-            g_option_context_free(options);
+        /* Allocate new parser */
+        g_option_context_free(options);
```

Comment:
If config file is no longer configurable from the the command line, there's no need to parse the command line twice anymore. Just load defaults from the config file and parse the command line. Once.

Reply:
Thanks. Cleaned up that unnecessary code.